package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "test": "cypress run --config video=false",
+    "ci": "concurrently --success first --kill-others \"yarn dev\" \"wait-on http://localhost:3000 && yarn test\""
```

Comment:
I think it's the second program (the running of tests) that should succeed :)

Reply:
I never used concurrently before, however, based on their docs: "Return exit code of zero or one based on the success failure of the "first" **child to terminate**" It's kinda misleading, the first to terminate are actually the tests, since the first process never terminates.

Reply:
Ah I didn't see that!